autonomi/README.md

```diff
@@ -9,9 +9,38 @@ Connect to and build on the Autonomi network.
 
 Add the autonomi crate to your `Cargo.toml`:
 
-```toml
-[dependencies]
-autonomi = { path = "../autonomi", version = "0.1.0" }
+```sh
+# `cargo add` adds dependencies to your Cargo.toml manifest file
+cargo add autonomi
+```
+
+### Example
+
+```rust
+use autonomi::{Bytes, Client, EvmNetwork, Wallet};
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Default wallet of testnet.
+    let key = "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80";
+
+    let client = Client::connect(&["/ip4/127.0.0.1/udp/1234/quic-v1".parse()?]).await?;
+    let wallet = Wallet::new_from_private_key(EvmNetwork::ArbitrumSepolia, key)?;
```

Comment:
```suggestion
    let wallet = Wallet::new_from_private_key(Default::default(), key)?;
```

Maybe we should use the default `EvmNetwork` here and add a separate section for setting the wallet up with a different/custom network?